tensorboard/plugins/histograms/histograms_demo.py

```diff
@@ -0,0 +1,88 @@
+# Copyright 2017 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Sample data exhibiting histogram summaries."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os.path
+
+from six.moves import xrange  # pylint: disable=redefined-builtin
+import tensorflow as tf
+
+# Directory into which to write tensorboard data.
+LOGDIR = '/tmp/histograms_demo'
+
+
+def run(logdir):
```

Comment:
Could you make this demo ascribe to the interface used by `scalars_demo` and `audio_demo`? The key method is `def run_all(logdir, verbose=False)`, where if `not verbose` then no output should be printed and if `verbose` you may print whatever you see fit. Then, `main` can call `run_all(LOGDIR, verbose=True)`, and our demo accumulator script can import the module and call `run_all(my_special_logdir, verbose=False)`.